activestorage/app/controllers/concerns/active_storage/file_server.rb

```diff
@@ -11,6 +11,7 @@ def serve_file(path, content_type:, disposition:)
           response.headers[name] = value
         end
 
+        response.headers.except!("X-Cascade", "x-cascade") if status == 416
```

Comment:
```suggestion
        response.headers.delete("X-Cascade", "x-cascade") if status == 416
```
Since we aren't using the return value, I think it makes sense to use Ruby `Hash#delete` over Rails `Hash#except!`

Other than that 👍 from me

Reply:
I wanted to make it one-liner, but `Hash#delete` accepts only one argument :(

Reply:
Ah, right. Maybe `response.headers.delete_if { |header| ["X-Cascade", "x-cascade"].include?(header) } if status == 416` ? 

I definitely agree `except!` looks the cleanest. I think if we keep `except!` we should add `require "active_support/core_ext/hash/except"` to ensure the extension is loaded

Reply:
Good catch! Added a missing require.